ansible/roles/host-lets-encrypt-certs-certbot/tasks/main.yml

```diff
@@ -4,12 +4,17 @@
   set_fact:
     _certbot_dir: "{{ _certbot_remote_dir }}/certbot"
 
+- name: Get home directory for {{ _certbot_user }} user
+  ansible.builtin.getent:
+    database: passwd
+    key: "{{ _certbot_user }}"
+
 - name: Check on AWS credentials
   when: _certbot_dns_provider is match('route53')
   block:
     - name: Verify if AWS Credentials exist on the host
       stat:
-        path: "/home/{{ _certbot_user }}/.aws/credentials"
+        path: "{{ ansible_facts.getent_passwd[_certbot_user][-2] + '/.aws/credentials' }}"
```

Comment:
Probably only philosophical but i would do instead: 

`ansible_facts.getent_passwd[_certbot_user][4] `

That's how i think backward compatibility could work. `getent` will never change the order but they might add more values